tests/xharness/Jenkins.cs

```diff
@@ -209,6 +210,8 @@ IEnumerable<TestData> GetTestData (RunTestTask test)
 				yield return new TestData { Variation = "AssemblyBuildTarget: SDK framework (debug, profiling)", MTouchExtraArgs = "--assembly-build-target=@sdk=framework=Xamarin.Sdk --assembly-build-target=@all=staticobject", Debug = true, Profiling = true, MonoNativeLinkMode = MonoNativeLinkMode.Static, MonoNativeFlavor = flavor };
 
 				yield return new TestData { Variation = "Release", MTouchExtraArgs = "", Debug = false, Profiling = false, MonoNativeLinkMode = MonoNativeLinkMode.Static };
+				yield return new TestData { Variation = "Release: SGenConc", MTouchExtraArgs = "", Debug = false, Profiling = false, MonoNativeLinkMode = MonoNativeLinkMode.Static, EnableSGenConc = true};
```

Comment:
Do the QA tests try all projects with the concurrent GC, or just some of them?

Reply:
It is tested in the following one:


- mini
- monotouch-test
- don’t link monotouch-test
- link sdk monotouch-test
- link all monotouch-test

- mscorlib
- System
- System.Core
- System.ServiceModel
- System.WebServices
- System.Runtime.Serialization
- System.Transactions
- System.Xml
- System.Xml.Linq
- Mono.Security
- Mono.Data.SqlLite
- System.Json
- System.Component.ModelDataAnnotations
- Mono.CSharp
- Mono.Data.Tds
- System.IO.Compression
- System.IO.Compression.FileSystem

And they do not make a diff in the tvOS Smoke + BCL Tests